utils/elftocpm65/elftocpm65.cc

```diff
@@ -1,23 +1,35 @@
 #include <fcntl.h>
-#include <unistd.h>
 #include <fmt/format.h>
-#include <libelf.h>
-#include <string>
 #include <fstream>
-#include <vector>
 #include <set>
+#include <string>
+#include <unistd.h>
```

Comment:
Neither `unistd.h` nor `elf.h` will be present on a MSVC install on Windows, so this utility is still POSIX-specific.

But, in retrospect, I don't want to burst your bubble, so I think that's OK, especially since it doesn't add anything to our install instructions on most POSIXen. Just add some CMake to prevent the build and install of this utility on Windows, and we can call it a day.

Reply:
Would it be feasible to just include a copy of elf.h? There's an Apple version which is CDDL, which (IIRC) should be compatible with Apache.

Reply:
I did a quick scan through the license; that should be fine. I think we're actually using MinGW, so there shouldn't be any compatibility problems. If there are, our builders will break, I'll roll it back, and we can fix things from there.